src/eq_fault_geom/geomio/cfm_shp_to_vtk.py

```diff
@@ -0,0 +1,214 @@
+from pathlib2 import Path
+import string
+import geopandas as gpd
+import pandas as pd
+from shapely.geometry import LineString
+import numpy as np
+import meshio
+# import pdb
+# pdb.set_trace()
+
+# Epsilon value.
+eps = 5000.0
+
+# Cell type.
+cell_type = 'triangle'
```

Comment:
I think this is only used inside the ..stirling.. function? 

Suggest we make it a function parameter (and set a default value), rather than a module-level varaible.

Reply:
cell_type is now a parameter passed to create_stirling_fault (and then passed along to create_mesh_from_trace).